js/src/views/ParityBar/parityBar.css

```diff
@@ -55,6 +55,7 @@
   overflow: auto;
   display: flex;
   background: rgba(0, 0, 0, 0.8);
+  min-height: 300px;
```

Comment:
Think it is actually because we have 2 min-heights in there somewhere. (pending container as well)